<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Convert GUI</class>
 <widget class="QMainWindow" name="Convert GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Convert GUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/icons.qrc">
    <normaloff>:/images/images/vscht.jpeg</normaloff>:/images/images/vscht.jpeg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>551</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frameActions">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>771</width>
       <height>121</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushResize">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/icons.qrc">
          <normaloff>:/images/images/resize.png</normaloff>:/images/images/resize.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>72</width>
          <height>72</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelResize">
        <property name="text">
         <string>Resize</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelRotate">
        <property name="text">
         <string>Rotate</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelCrop">
        <property name="text">
         <string>Crop</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelFilter">
        <property name="text">
         <string>Add filter</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="labelText">
        <property name="text">
         <string>Add text</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="labelBlur">
        <property name="text">
         <string>Blur</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="pushBlur">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/icons.qrc">
          <normaloff>:/images/images/blur.png</normaloff>:/images/images/blur.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>72</width>
          <height>72</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pushText">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/icons.qrc">
          <normaloff>:/images/images/addtext.png</normaloff>:/images/images/addtext.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>72</width>
          <height>72</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushFilter">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/icons.qrc">
          <normaloff>:/images/images/filter.png</normaloff>:/images/images/filter.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>72</width>
          <height>72</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushCrop">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/icons.qrc">
          <normaloff>:/images/images/crop.png</normaloff>:/images/images/crop.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>72</width>
          <height>72</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushRotate">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/icons.qrc">
          <normaloff>:/images/images/rotate.png</normaloff>:/images/images/rotate.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>72</width>
          <height>72</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="labelPreview">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;File(s) preview&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushExport">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>280</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Export/Convert</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboFormat">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>250</y>
       <width>86</width>
       <height>21</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>BMP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GIF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>JPG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PBM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PDF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PGM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PNG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PPM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XBM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XPM</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelFormat">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>250</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Format:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushSelect">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>470</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Select file(s)</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditLog">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>470</y>
       <width>381</width>
       <height>71</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelLog">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>450</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Progress log&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="previewStackedWg">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>281</width>
       <height>261</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageNoFile">
      <widget class="QFrame" name="frameNoFile">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>261</width>
         <height>251</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="labelNoFile">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>90</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;No file selected&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="pageSingleFile">
      <widget class="QLabel" name="labelPicture">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>271</width>
         <height>241</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pageMultiple">
      <widget class="QListWidget" name="listOfFiles">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>271</width>
         <height>221</height>
        </rect>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="unselectAll">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>0</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Unselect all</string>
       </property>
      </widget>
      <widget class="QPushButton" name="selectAll">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Select all</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QCheckBox" name="singlePDFcheck">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>380</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Export as a single PDF file</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExport"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open (Ctrl+O)</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export (Ctrl+S)</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit (Ctrl+Q)</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo (Ctrl+Z)</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo (Ctrl+Shift+Z)</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
